draft-ietf-mls-architecture.md

```diff
@@ -425,125 +428,105 @@ cryptographic information:
 * A credential from the Authentication Service attesting to the
   binding between the identity and the client's signature key.
 
-* The client's asymmetric encryption public key material.
+* The client's asymmetric encryption public key.
 
 All the parameters in the KeyPackage are signed with the signature private key
 corresponding to the credential.
 
-As noted above, users may own multiple clients, each with their
-own keying material, and thus there may be multiple entries
-stored by each user.
-
-The Delivery Service is also responsible for allowing users to add,
-remove or update their initial key material, and for ensuring
-that the identifier for these keys are unique across all keys
-stored on the Delivery Service.
```

Comment:
Removed because this seems to be redundant with paragraph 1 of this section + the last sentence (DS ensures uniqueness) is not true